getnacelle/nacelle-js
# chore(gatsby): product content resolvers

### WHY are these changes introduced?

Addresses 
 - [ENG-5731](https://nacelle.atlassian.net/browse/ENG-5731)





### WHAT is this pull request doing?



- Removes secondary fetch to resolve referenced products `utils/resolvers`
- Refactors `gatsby-config.js` to import `createPages` and `createResolvers`
- Uses `createResolvers` to allow for querying referenced products in initial queries.

## 

### Testing
- `git checkout ENG-5731/gatsby-product-content-resolvers`
- `cd references-stores/gatsby`
- delete `node_modules/@nacelle/gatsby-source-nacelle`
- `npm install`
- `npm run clean && npm run develop`
- See products displayed in `FeaturedProducts` section on homepage, and  in `CartCrosssells` component in cart
